src/main/java/org/privacyidea/Endpoint.java

```diff
@@ -15,6 +30,7 @@
 import okhttp3.Request;
 import okhttp3.Response;
 
+import static org.privacyidea.PIConstants.*;
```

Comment:
If You import `*` do You still need the other imports from `PIConstants`?

Reply:
good point. Importing ``*`` is actually bad style. Will fix it

Reply:
Since there are just constants in this namespace i think this should be ok.